BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -170,11 +254,12 @@ public final class ConvivaAnalytics: NSObject {
         client.updateContentMetadata(sessionKey, metadata: contentMetadata)
     }
 
-    private func endSession() {
+    private func internalEndSession() {
         client.detachPlayer(sessionKey)
         client.cleanupSession(sessionKey)
         client.releasePlayerStateManager(playerStateManager)
         sessionKey = NO_SESSION_KEY
+        metadataOverrides = MetadataOverrides()
```

Comment:
What do you think about having this in the `buildContentMetadata` or `internalInitSession`?

Reply:
ok then 👍